MergeAccountingClient/model_utils.py

```diff
@@ -103,59 +103,61 @@ def composed_model_input_classes(cls):
 class OpenApiModel(object):
     """The base class for all OpenAPIModels"""
 
-    def set_attribute(self, name, value):
-        # this is only used to set properties on self
+def set_attribute(self, name, value):
+    # this is only used to set properties on self
 
-        path_to_item = []
-        if self._path_to_item:
-            path_to_item.extend(self._path_to_item)
-        path_to_item.append(name)
-
-        if name in self.openapi_types:
-            required_types_mixed = self.openapi_types[name]
-        elif self.additional_properties_type is None:
-            raise ApiAttributeError(
-                "{0} has no attribute '{1}'".format(
-                    type(self).__name__, name),
-                path_to_item
-            )
-        elif self.additional_properties_type is not None:
-            required_types_mixed = self.additional_properties_type
-
-        if get_simple_class(name) != str:
-            error_msg = type_error_message(
-                var_name=name,
-                var_value=name,
-                valid_classes=(str,),
-                key_type=True
-            )
-            raise ApiTypeError(
-                error_msg,
-                path_to_item=path_to_item,
-                valid_classes=(str,),
-                key_type=True
-            )
+    path_to_item = []
+    if self._path_to_item:
+        path_to_item.extend(self._path_to_item)
+    path_to_item.append(name)
```

Comment:
are these changes intentional, or due to a generator update?

Reply:
Oh wow great catch, I think I might have an error in one of my templates that overrides the base openapi generator template.